xamarin/Xamarin.Forms
# [UWP] Fixes IsTabStop property in Layouts

### Description of Change ###

Fixes IsTabStop property in Layouts

### Issues Resolved ### 


- fixes #4653

### API Changes ###

 
 None

### Platforms Affected ### 


- UWP

### Behavioral/Visual Changes ###


None

### Before/After Screenshots ### 


Not applicable

### Testing Procedure ###

- run UItest 4653
- check that the keyboard can not select any control inside grid.
- click on `Toggle Grid IsTabStop`
- tab should work on some buttons, entry
- the ScrollView does not gain focus anyway

### PR Checklist ###
